### PR TITLE
Network peer updates

### PR DESCRIPTION
**Pull request by roman-khimov**
This one updates broadcasting a little giving more priority for everything related to consensus.

### CODE REVIEW

File: pkg/network/server.go [hunk 1]
```diff
@@ -676,25 +679,44 @@ func (s *Server) requestTx(hashes ...util.Uint256) {
...
 
-func (s *Server) relayInventoryCmd(cmd CommandType, t payload.InventoryType, hashes ...util.Uint256) {
-	payload := payload.NewInventory(t, hashes)
-	msg := s.MkMsg(cmd, payload)
-
+// iteratePeersWithSendMsg sends given message to all peers using two functions
+// passed, one is to send the message and the other is to filtrate peers (the
+// peer is considered invalid if it returns false).
+func (s *Server) iteratePeersWithSendMsg(msg *Message, send func(Peer, []byte) error, peerOK func(Peer) bool) {
+	pkt, err := msg.Bytes()
```
Comment (volekerb) [inline pkg/network/server.go]: nit: why pkt?

Comment (roman-khimov) [reply pkg/network/server.go]: Well, initially it was just `b`, maybe something reminding of a packet is a bit better.

File: pkg/network/server.go [hunk 2]
```diff
@@ -676,25 +679,44 @@ func (s *Server) requestTx(hashes ...util.Uint256) {
...
+	// even though it's getdata.
+	s.broadcastHPMessage(msg)
 }
 
-func (s *Server) relayInventoryCmd(cmd CommandType, t payload.InventoryType, hashes ...util.Uint256) {
-	payload := payload.NewInventory(t, hashes)
-	msg := s.MkMsg(cmd, payload)
-
+// iteratePeersWithSendMsg sends given message to all peers using two functions
+// passed, one is to send the message and the other is to filtrate peers (the
```
Comment (fyrchik) [inline pkg/network/server.go]: filtrate -> filter

Comment (roman-khimov) [reply pkg/network/server.go]: Well, both are correct, so I'm sticking with the first one written.

**Pull request merged**